### PR TITLE
Add `!introreminder` command

### PR DESCRIPTION
**Pull request by jmariondev**
This admin-only command DMs every user on the server without any roles and reminds them to introduce themselves in #welcome-room.

Closes gaymers-discord/DiscoBot#194

Per @brandon-a 😘

### CODE REVIEW

File: commands/introreminder.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+const REMINDER_MESSAGE = 'Hi! :smile:\n\n' +
+  'This is just a friendly reminder that once you introduce yourself in ' +
+  '#welcome-room on the Gaymers Discord server, you\'ll be given access to ' +
+  'the rest of our channels!\n\n' +
+  'Some examples of things you can say about yourself are: the games that ' +
+  'you play, where you\'re from, and how you heard about us.\n\n' +
+  'I hope to see you soon! :heart:\n\n' +
+  '(Users that don\'t introduce themselves in 24 hours are automatically ' +
+  'kicked. If you introduced yourself but haven\'t gotten staff attention ' +
+  'yet, feel free to ping staff with `@Moderator`)';
```
Comment (brandon-a) [inline commands/introreminder.js:30]: Maybe we can change this to 

```
 +const REMINDER_MESSAGE = 'Hi! :smile:\n\n' +
 +  'This is just a friendly reminder that you now have 24 hours to introduce yourself in ' +
 +  '#welcome-room on the Gaymers Discord server, you\'ll then be given access to ' +
 +  'the rest of our channels!\n\n' +
 +  'Some examples of things you can say about yourself are: the games that ' +
 +  'you play, where you\'re from, and how you heard about us.\n\n' +
 +  'I hope to see you soon! :heart:\n\n' +
 +  '(Users that don\'t introduce themselves in 24 hours are automatically ' +
 +  'kicked. If you introduced yourself but haven\'t gotten staff attention ' +
 +  'yet, feel free to ping staff with `@Moderator`)';
```

**Review by brandon-a — CHANGES_REQUESTED** [review]
Maybe we can make the one change I suggested to the wording, I wanted to make the message more stern so that people don't skim over it thinking there are no consequences to taking no action.

**Comment by jmariondev** [comment]
@brandon-a How about this? I made what you said a little softer but still with the same gist.

**Review by brandon-a — APPROVED** [review]
Looks good!

**Review by otter-computer — CHANGES_REQUESTED** [review]
I don't see any code that kicks people, is this planned?

Also, this would be much better run as a cron with either in-memory or database logging of who has been reminded when.

@brandon-a feel free to merge if you want this already but it would be much better implemented if it was automatic.

**Comment by otter-computer** [comment]
Just to add to this, this message would be much better if sent in the server it relates to - i.e. in the welcome room. It'd be a little messier, but at least people will instantly know what server they need to reply to (and honestly it's just welcome room rite?)

**Comment by jmariondev** [comment]
Yeah, @brandon-a and I talked about the implementation of kicking as well. I agree that this would actually better be implemented as a cron. I think at that point it would be best to back it with a db datastore instead of RAM.

**Comment by jmariondev** [comment]
This is pending a decision in #256.

**Pull request closed without merging**